features/manage_response_sets.feature

```diff
@@ -47,7 +48,7 @@ Feature: Manage Response Sets
     And I click on the option to Extend the Response Set with the name "Gender Full"
     And I fill in the "Name" field with "Gender Partial"
     And I fill in the "Description" field with "M / F / O"
-    And I click on the "Create Response set" button
```

Comment:
Does this mean we no longer have a test of initial creation? We should have tests for both.